### ISSUE TITLE
jpype error when fetching variables

### ISSUE THREAD

**Issue by Pingul** [rdemaria/pytimber#47]
I think I've found a bug. I receive an error when fetching variables:

> java.lang.ArithmeticExceptionPyRaisable: java.lang.ArithmeticException: Can't convert long to double. Use long representation instead.

It seems to be a conversion issue somewhere in `jpype`. This is the first time it happens for me, so it seems to be quite rare. 

Here is a reproducible case:

    In [6]: import pytimber
    In [7]: db = pytimber.LoggingDB()
    In [8]: meta = db.getLHCFillData(5668)
    In [9]: var = ['TCDQA.A4L6.B2:MEAS_MOTOR_LD', 'TCDQA.A4L6.B2:MEAS_MOTOR_LU', 'TCDQA.A4L6.B2:MEAS_MOTOR_RD', 'TCDQA.A4L6.B2:MEAS_MOTOR_RU', 'TCDQA.A4L6.B2:MEAS_PROFILE_TIME']
    In [10]: ramp_mode = next((item for item in meta['beamModes'] if item['mode'] == "RAMP"), None)
    In [11]: db.getAligned(var, ramp_mode['startTime'], ramp_mode['endTime'])
    ---------------------------------------------------------------------------
    java.lang.ArithmeticExceptionPyRaisable   Traceback (most recent call last)
    <ipython-input-14-e0af2ad64f2c> in <module>()
    ----> 1 db.getAligned(var, ramp_mode['startTime'], ramp_mode['endTime'])

    /opt/local/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pytimber/pytimber.py in getAligned(self, pattern_or_list, t1, t2, fundamental, master, unixtime)
        275             self._log.info('{0} seconds for aqn'.format(time.time()-start_time))
        276             out[v] = self.processDataset(
    --> 277                 res, res.getVariableDataType().toString(), unixtime
        278             )[1]
        279         return out

    /opt/local/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pytimber/pytimber.py in processDataset(self, dataset, datatype, unixtime)
        182                 val = np.array(tt.getStringValues(), dtype='U')
        183             elif datatype == 'NUMERIC':
    --> 184                 val = tt.getDoubleValue()
        185             elif datatype == 'FUNDAMENTAL':
        186                 val = 1

    java.lang.ArithmeticExceptionPyRaisable: java.lang.ArithmeticException: Can't convert long to double. Use long representation instead.

**Comment by levens** [comment]
The variable "TCDQA.A4L6.B2:MEAS_PROFILE_TIME" seems to be the one causing the problem...

**Comment by levens** [comment]
OK, it seems that tt can either be of type:

**Comment by levens** [comment]
We could do something like this:

```python
if tt.getClass() == db._jpype.JPackage('cern').accsoft.cals.extr.domain.core.timeseriesdata.spi.NumericLongData:
    val = tt.getLongValue()
elif tt.getClass() == db._jpype.JPackage('cern').accsoft.cals.extr.domain.core.timeseriesdata.spi.NumericDoubleData:
    val = tt.getDoubleValue()
else:
    raise TypeError
```

But, it's a little ugly...

**Comment by rdemaria** [comment]
It is boring but I would handle all cases...

**Comment by levens** [comment]
OK, this fix is applied in #48
